test.sh

```diff
@@ -110,7 +110,7 @@ vagrantDestroy
 nodesOff
 
 # Power on vagrant box and nodes 
-vagrantUp
+vagrantUp "RackHD/packer/ansible/roles/monorail/files/auth_config"
```

Comment:
OK,  understood.  Your test toggles auth enable/disabled configs.  That works.  Howerver,  with auth enabled and both NB and SB routes split,  the integration tests may fail if it cannot access some SB routes from the NB interface.  Have you tested that case?

+1 on the config.json. Thanks.